bentoml/handlers/image_handler.py

```diff
@@ -128,29 +128,37 @@ def handle_request(self, request, func):
             response object
         """
         if request.method != "POST":
-            return Response(response="Only accept POST request", status=400)
+            return Response(
+                response="BentoML#ImageHandler only accept POST request", status=400
+            )
 
-        input_files = [request.files.get(filename) for filename in self.input_names]
-        if len(input_files) == 1 and input_files[0] is None:
```

Comment:
There is one case we tried to cover previously:
if a user submits only one file, even without an incorrect filename, we can still process it.

Reply:
got you, good catch!

Reply:
Yes.  I believe that is the case, when you submit only one file but with incorrect filename.  It looks like @parano is handling reading data from files with incorrect file names.